release-notes/major11/11.1.0/valtimo-backend-libraries.md

```diff
@@ -0,0 +1,44 @@
+# Backend libraries 11.1.0
+
+## New Features
+
+The following features were added:
+
+* **Form.io forms targetKey property**
+
+  Form.io forms in Valtimo now support a `targetKey` property. This property defines a location to which the submitted
+  form values should be stored.
+
+* **New feature2**
+
+  Description of the new feature goes here.
+  Also link to the page that explains the feature in greater detail.
+
+
+## Bugfixes
+
+The following bugs were fixed:
+
+* **Fix DMN upload**
+
+  The uploading of DMN files through the endpoint `/v1/process/definition/deployment` has been fixed.
```

Comment:
Can we describe what has been fixed better? I see the FE release notes are more elaborate on this.